ovs-service/src/main/java/org/dcsa/ovs/mapping/ServiceScheduleMapper.java

```diff
@@ -0,0 +1,12 @@
+package org.dcsa.ovs.mapping;
+
+import org.dcsa.ovs.persistence.entity.Service;
+import org.dcsa.ovs.transferobjects.ServiceScheduleTO;
+import org.mapstruct.Mapper;
+
+@Mapper(
+  componentModel = "spring",
+  uses = {VesselScheduleMapper.class})
+public interface ServiceScheduleMapper {
```

Comment:
Since mapstruct only maps fields that have exactly the same name, maybe put in a FieldValidator test on this

Reply:
Added.